news/2020-06-26-mappers-report-May-June.md

```diff
@@ -0,0 +1,201 @@
+
+### osu!mania
+
+- **[Chord usage at 1/4 pattern in normal difficulties](https://osu.ppy.sh/community/forums/topics/1079778):** This change will allow to use chords as well at the beginning of 1/4 pattern in Normal and 1/2 pattern in Easy difficulties. Was only to end a pattern before this change.
+
+## Open Ranking Criteria Proposals
+
+There are various open Ranking Criteria proposal in the [forum](https://osu.ppy.sh/community/forums/87). Many of them are listed in the last issue of the [Mappers' Report](https://osu.ppy.sh/home/news/2020-05-11-mappers-report-april#rc-changes), but we also got some new one which you can find below!
+
+For all Modes:
+
+- **[Discussion to fix Vetoes](https://osu.ppy.sh/community/forums/topics/1068618)**
+- **[Increase average bitrate limitation from 192kbs to 320kbs](https://osu.ppy.sh/community/forums/topics/1072965)**
+- **[Spoiler in maps](https://osu.ppy.sh/community/forums/topics/1083667)**
+
+For osu!:
+
+- **[Make a limit for longest combo in osu!std&CTB map](https://osu.ppy.sh/community/forums/topics/1071402)**
+
+For osu!mania:
+
+- **[change chord guidelines in 4k normal/hard diffs](https://osu.ppy.sh/community/forums/topics/1067554)**
+- **[Redefine the "stream" terminology in mania RC](https://osu.ppy.sh/community/forums/topics/1079647)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+The Beatmap Nominators never rest, the Beatmap Nominators never sleep. Just kidding, of course they do, they are still humans. However, the group size is always changing! New are coming, old are going. This month was no exception!
+
+### BN Additions
+
+- **osu!:** [Teky](https://osu.ppy.sh/users/10520912), [DeviousPanda](https://osu.ppy.sh/users/4966334), [Stixy](https://osu.ppy.sh/users/9000308), [Trynna](https://osu.ppy.sh/users/2652951), [Fall](https://osu.ppy.sh/users/4800816), [Elayue](https://osu.ppy.sh/users/6400861), [NexusQI](https://osu.ppy.sh/users/13822800) & [Garden](https://osu.ppy.sh/users/2849992)
+- **osu!taiko:** [realy0\_](https://osu.ppy.sh/users/8863359), [\_DUSK\_](https://osu.ppy.sh/users/6092181), [davidminh0111](https://osu.ppy.sh/users/9623142), [Cychloryn](https://osu.ppy.sh/users/6921736) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Baron](https://osu.ppy.sh/users/10286499), [Secre](https://osu.ppy.sh/users/2306637) & [Bastian](https://osu.ppy.sh/users/6345176)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107), [Shima Rin](https://osu.ppy.sh/users/6089608), [Garalulu](https://osu.ppy.sh/users/757783) & [Muse Dash](https://osu.ppy.sh/users/13695676)
+
+Do you want to see your own name on that list too? And do you like modding and helping other people with their beatmaps? Then become a Beatmap Nominator! The applications are always open. Head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!
```

Comment:
dunno why but first three sentences sound incorrect to me, although I have no idea how to reword them properly

Reply:
Hmm, can't really see what exactly sounds wrong on it?

Reply:
They sound alright to me but maybe they'd be a little better-sounding this way?

```suggestion
Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of those questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. Head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!
```

Reply:
Okay yeah, sounds indeed a bit better. Gonna use that!